Automattic/jetpack
# My Jetpack: Add Social card

Mostly the same as  https://github.com/Automattic/jetpack/pull/2413. After resolving conflicts on that PR, I removed a function that I think is not necessary. After resolving conflicts in the linked PR, the pre-commit phpcs checks kept giving me OOM's, so I decided to make a new PR. 

#### Changes proposed in this Pull Request:

*  Made changes to add the "Jetpack Social" product to My Jetpack.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No
#### Testing instructions:






*  Have another stand-alone plugin installed such as boost or backup.
*  cd-into the plugin's directory and do `composer require automattic/jetpack-my-jetpack` (Without this step, the new classes created in the my jetpack src folder, weren't getting picked up).
*  Go to My Jetpack and notice Jetpack Social. 
*  Activate Social. 
*   Deactivate Social and come back to the My Jetpack screen to notice that the Jetpack Social plugin shows as deactivated.


<img width="1186" alt="image" src="https://user-images.githubusercontent.com/746152/169815228-ddbf966e-d33a-4fae-9be9-072223719215.png">
